src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamespaceSymbol.cs

```diff
@@ -190,6 +191,21 @@ protected void LoadAllMembers(IEnumerable<IGrouping<string, TypeDefinitionHandle
             LazyInitializeTypes(nestedTypes);
         }
 
+        private int GetQualifiedNameLength()
+        {
+            int length = this.Name.Length;
+
+            var parent = ContainingNamespace;
+            while (parent?.IsGlobalNamespace != true)
```

Comment:
If `parent` is ever null, won't we get a NullReferenceException at line 203? Consider changing this to ```while (!parent.IsGlobalNamespace)``` so it's clear `parent` is non-null.